contracts/multisig-prover/src/contract.rs

```diff
@@ -298,7 +298,7 @@ mod tests {
 
     fn query_get_verifier_set(
         deps: Deps,
-    ) -> Result<Option<VerifierSet>, axelar_wasm_std::ContractError> {
+    ) -> Result<Option<(String, VerifierSet)>, axelar_wasm_std::ContractError> {
```

Comment:
hey CJ, the ID is actually a String (https://github.com/axelarnetwork/axelar-amplifier/blob/main/contracts/multisig/src/verifier_set.rs#L52-L54) and the query to get the verifier set in `Multisig` expects a String as well (https://axelarnetwork.atlassian.net/browse/AXE-4182) since it's stored as such. I'm not sure why it was done this way or if it should be changed, but in this case since it's a query I think we should return whatever the underlying type is.

Reply:
hmm yeah, I think it would be preferable to change it to a Hash type, but not for this PR